utils/script-python.h

```diff
@@ -14,6 +14,8 @@ struct script_info;
 
 #if defined(HAVE_LIBPYTHON2) || defined(HAVE_LIBPYTHON3)
 
+#undef _XOPEN_SOURCE
+#undef _POSIX_C_SOURCE
```

Comment:
Right! This header file activated when `configure` script `have_libpython*` condition from existing user's `libpython`.
Then, `check-deps/Makefile` -> `HAVE_LIBPYTHON*` macro would be enabled.